ci-operator/step-registry/sdn/conf/single-manifest/sdn-conf-single-manifest-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: sdn-conf-single-manifest
```

Comment:
same as multitenant comment maybe change to sdn-conf-singletenant-manifest

Reply:
Again shorter filenames